Sources/Private/LayerContainers/Utility/CoreTextRenderLayer.swift

```diff
@@ -122,12 +123,10 @@ final class TextLayer: CALayer {
 
     ctx.textMatrix = .identity
     ctx.setAllowsAntialiasing(true)
-    ctx.setAllowsFontSmoothing(true)
```

Comment:
No clue why, but removing this fixes #1488. All of the text examples in the regression suite still render just fine without this.

Reply:
I tested a few different animations with `TextLayer`s in the macOS example app and didn't see any differences with this change.